plasmapy/classes/sources/openpmd_hdf5.py

```diff
@@ -166,3 +165,8 @@ def is_datasource_for(cls, **kwargs):
                 openPMD = False
 
         return openPMD
+
+
+class DataStandardError(Exception):
+    """An exception for when HDF5 is not defined by OpenPMD standard."""
```

Comment:
Perhaps we could revise the docstring for `DataStandardError` so that the definition is broader.  Maybe something like `"""An exception for when the structure of a data file does not match the expected standard."""?

Reply:
Works for me!